Kudu.Services/ServiceHookHandlers/GithubHandler.cs

```diff
@@ -1,10 +1,29 @@
 ﻿using System.Web;
 using Newtonsoft.Json.Linq;
+using Kudu.Core;
+using System;
+using Kudu.Core.Infrastructure;
+using System.IO;
 
 namespace Kudu.Services.ServiceHookHandlers
 {
     public class GitHubHandler : GitHubCompatHandler
     {
+        private IEnvironment environmentInject { get; set; } = null;
```

Comment:
Casing: `EnvironmentInject`. However, for something like this we typically use private fields and not props. You can just call it `_environment`.